cmd/frontend/internal/cli/serve_cmd.go

```diff
@@ -21,6 +21,8 @@ import (
 	"go.opentelemetry.io/otel"
 
 	oce "github.com/sourcegraph/sourcegraph/cmd/frontend/oneclickexport"
+	"github.com/sourcegraph/sourcegraph/cmd/worker/shared/init/codeintel"
```

Comment:
```suggestion
	"github.com/sourcegraph/sourcegraph/internal/codeintel"
```

Generally, `cmd/frontend` shouldn't import code from `cmd/worker`. I moved the init logic into a shared package that should be clean to use, though.